.github/workflows/codeql.yml

```diff
@@ -21,7 +22,7 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
```

Comment:
You could have dependabot take care of these.
See https://github.com/vapor/penny-bot/blob/main/.github/dependabot.yml.
You can set the interval to be `weekly` at a specific day and a specific hour too.

Reply:
Hadn't really been keeping up with this, anyway copied your dependabot setup and removed codeql.yml

Reply:
👍

The Swift part though is unnecessary because this library doesn't contain (or need to contain) any `Package.resolved` files 🙂:
```yml
- package-ecosystem: "swift"
    directory: "/"
    schedule:
      interval: "daily"
    open-pull-requests-limit: 6
    allow:
      - dependency-type: all
    groups:
      all-dependencies:
        patterns:
          - "*"
```

Reply:
Also `enable-beta-ecosystems: true` is unnecessary too now, although shouldn't hurt, i thought i've already removed that from Penny's. 😅